lib/openstudio/extension.rb

```diff
@@ -72,15 +71,15 @@ def list_measures
       # Relevant files such as weather data, design days, etc.
       # return the absolute path of the files or nil if there is none, can be used when configuring OSWs
       def files_dir
-        return File.absolute_path(File.join(File.dirname(__FILE__), '../files/'))
+        return File.absolute_path(File.join(File.dirname(__FILE__), '..', 'files'))
```

Comment:
@nllong is this a rubocop thing?  I had started using ```root_dir``` instead of File.dirname(__FILE__) too

Reply:
I don't think this is a rubocop thing. Since you are using File.join, you should not specify the slashes since those are platform dependent.